components/loginPg/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Aurobindo Gupta" />
+  <meta name="description" content="Personal Ecommerce Website" />
+  <link rel="icon" type="image/x-icon" href="Utils/images/iconpanda16.ico" />
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="https://panda-ui-neog.netlify.app/rootStyles.css" />
+  <link rel="stylesheet" href="/styles.css" />
+  <link rel="stylesheet" href="styles.css" />
+  <title>Ecommerce</title>
+</head>
+
+<body>
+  <!-- ................BASE CONTAINER............. -->
+  <div class="baseContainer">
+    <!-- ................NAV BAR............. -->
+
+    <nav class="navBar">
+      <div class="heroContainer">
+        <p class="heroTitle">MyWebsite</p>
+      </div>
+      <div class="navBar-searchBar">
+        <form class="searchBar">
+          <input class="searchBar-input" placeholder="Search" />
+        </form>
+      </div>
+      <ul class="navLinks">
+        <li class="navLinksBtn">
+          <button class="btn btn-secondary">Login</button>
+        </li>
+        <li class="navLinksBtn" onclick="window.location.href=''">
```

Comment:
Please remove onClick if not needed. Or use a JavaScript file separately.

Reply:
done